.eslintrc.js

```diff
@@ -9,4 +9,15 @@ module.exports = {
     node: true,
     jest: true,
   },
+  rules: {
```

Comment:
To nie jest Ci chyba potrzebne 😉?

Reply:
własnie to jest dziwne tutaj, bo w Dish Schema np. 
`@Prop(MacroSchema) 
carbohydrates: {}`

To mam mi eslint krzyczy. Więc może wywale po prostu to co dopisalem, a i dam zamiast {} po prostu object